### PR TITLE
DFAST-85: Refactor existing unit tests & unit test names for Kernel

### PR DESCRIPTION
**Pull request by MRVermeulenDeltares**

### CODE REVIEW

File: tests/test_kernel.py [hunk 1]
```diff
@@ -1,292 +1,341 @@
...
+                numpy.array([0.06048387096774193, 0.09274193548387097, 0.09274193548387097, 1., 0.])]
 
-        assert (abs(zgemc - zgem) < 1e-13).all() and (abs(zmax - zmaxc) < 1e-13).all() and ((zmin - zminc) < 1e-13).all() == True
+        zgemc, zmaxc, zminc, dzb = dfastmi.kernel.main_computation([dzq1, dzqS, dzq2, dzq3], T, rsigma)
+        
+        self.print_values_floatmode_unique(zgem, zgemc, zmax, zmaxc, zmin, zminc, zdzb, dzb)   
 
+        assert (abs(zgemc - zgem) < 1e-13).all()
+        assert (abs(zmax - zmaxc) < 1e-13).all()
+        assert ((zmin - zminc) < 1e-13).all()
```
Comment (ralph-peelen-deltares) [inline tests/test_kernel.py:316]: double compare floating point, use abs.

File: tests/test_kernel.py [hunk 2]
```diff
@@ -1,292 +1,341 @@
...
+        
+        assert ((abs(zdzb[0]-dzb[0])) < 1e-13).all()
+        assert ((abs(zdzb[1]-dzb[1])) < 1e-13).all()
+        assert ((abs(zdzb[2]-dzb[2])) < 1e-13).all()
+        
+    def test_given_stagnant_t_when_main_computation_then_expect_returned_values_are_in_range_of_expected_values(self):
+        """
+        Testing main_computation.
```
Comment (ralph-peelen-deltares) [inline tests/test_kernel.py:234]: comment weg volgens mij

Comment (MRVermeulenDeltares) [reply tests/test_kernel.py]: done

File: tests/test_kernel.py (hunk 1 above)
Comment (MRVermeulenDeltares) [reply tests/test_kernel.py]: done

**Pull request merged**